### PR TITLE
Add support for the sass-lint linter

### PR DESCRIPTION
**Pull request by alexBaizeau**
More details here:

https://github.com/sasstools/sass-lint

### CODE REVIEW

File: tests/tools/test_sasslint.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        eq_([], self.problems.all(self.fixtures[0]))
+
+    @needs_sasslint
+    def test_process_files__one_file_fail(self):
+        self.tool.process_files([self.fixtures[1]])
+        problems = self.problems.all(self.fixtures[1])
+        eq_(1, len(problems))
+
+        fname = self.fixtures[1]
+        expected = Comment(fname, 4, 4, "Mixins should come before declarations (mixins-before-declarations)")
```
Comment (alexBaizeau) [inline tests/tools/test_sasslint.py]: Pretty funny, that this is reviewed by itself

Comment (markstory) [inline tests/tools/test_sasslint.py]: :smile:

**Comment by markstory** [comment]
Build failures look relevant.

**Comment by alexBaizeau** [comment]
@markstory I fixed up the build failure by pinning node to v5 in the travis worker, not ideal but it works

**Comment by markstory** [comment]
0.11.0 tagged and published.

**Pull request merged**